bundles/org.openhab.ui.basic/snippets-src/main_static.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
```

Comment:
Why isn't it detected by Git as a file moving, like for main.html ?

Reply:
I wonder about that too

Reply:
Are you sure you did not change file encoding for example ?

Reply:
I reset the commits and cleaned it up that way. It now shows up as moved.